### DISCUSSION TITLE
Splash screen should show more diversity

### DISCUSSION THREAD

**Discussion by DanMcM-SD** [dbeaver/dbeaver#11741] [General]
I believe an application with the title "community" should show a more diverse collection of faces. It would be nice if the dbeaver splash screen reflected our community better.

**Comment by serge-rider** [comment]
Previously we never looked at it from that angle. 
I hope it didn't offend anyone.

In any case we changed product splashscreens in version 21.

**Reply by DanMcM-SD** [reply]
Fair enough. I am not offended or suggesting offense, I just think it would look better and be viewed as better if the images of the people are more diverse.

You indicate the screen changed in "version 21". I am on version 7.3.5.202102151746. Are we talking about two different products or two different versioning methods?

**Comment by hydrandt** [comment]
The new splash screen is awesome!:) is there a story behind it?

**Comment by diegosiltel** [comment]
Hi there, awesome logo, only a personal doubt: why the elephant ...? is by the simil between hadoop and postgre logos, the beaver and the octocat is clearly intuitive
